docs/fsharp/tutorials/getting-started/getting-started-vscode.md

```diff
@@ -1,4 +1,278 @@
+
+## Prerequisites
+
+F# 4.0 or higher must be installed on your machine to use Ionide.
+
+### Windows
+
+If you're on Windows, the easiest way to do that is to install [Visual Studio with the Visual F# Tools](getting-started-visual-studio.md#installing-the-visual-f#-tools).  This will install all of the necessary components to write and execute F# code.
+
+If you prefer not to install Visual Studio, use the following instructions:
+
+1. Install [.NET Framework 4.5 or higher](https://www.microsoft.com/en-US/download/details.aspx?id=30653) if you're running Windows 7.  If you're using Windows 8 or higher, you do not need to do this.
+
+2. Install the Windows SDK for your OS:
+
+    * [Windows 10 SDK](https://dev.windows.com/en-US/downloads/windows-10-sdk)
+    * [Windows 8.1 SDK](http://msdn.microsoft.com/windows/desktop/bg162891)
+    * [Windows 8 SDK](http://msdn.microsoft.com/windows/hardware/hh852363.aspx)
+    * [Windows 7 SDK](http://www.microsoft.com/download/details.aspx?id=8279)
+
+3. Install the [Microsoft Build Tools 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48159).  You may also need to install [Microsoft Build Tools 2013](https://www.microsoft.com/en-us/download/details.aspx?id=40760).
+
+4. Install the [Visual F# Tools](https://www.microsoft.com/en-us/download/details.aspx?id=48179).
+
+On 64-bit Windows, the compiler and tools are located here:
+
+```
+C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\fsc.exe
+C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\fsi.exe
+C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\fsiAnyCpu.exe
+```
+
+On 32-bit Windows, the compiler tools are located here:
+
+```
+C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsc.exe
+C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsi.exe
+C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsiAnyCpu.exe
+```
+
+Ionide automatically detects the compiler and tools, but if it doesn't for some reason, you can manually add the containing folder (`...\Microsoft SDKs\F#\4.0`) to your PATH.
+
+### macOS
+
+On macOS, Ionide uses Mono.  The easiest way to install Mono on macOS is via Homebrew.  Simply type the following into your terminal:
+
+```
+brew install mono
+```
+
+### Linux
+
+On Linux, Ionide also uses Mono.  If you're on Debian or Ubuntu, you can use the following:
+
+```
+sudo apt-get update
+sudo apt-get install mono-complete fsharp
+```
+
+## Installing Visual Studio Code and the Ionide plugin
+
+You can install Visual Studio Code on the [code.visualstudio.com](https://code.visualstudio.com) website.  After that, there are two ways to find the Ionide plugin:
+
+1. Use the Command Palette (Ctrl-P on Windows, ⌘-P on macOS, Ctrl-Shift-P on Linux) and type the following:
```

Comment:
Now that I saw the key bindings on VS Code docs, I really want to see that supported in OPS too: 
https://code.visualstudio.com/docs/editor/codebasics#_command-palette

I think the correct way to show this would be to says Ctrl+Shift+P with the plus sign and not a dash. Also, for Windows, the right keyboard combination is also Ctrl+Shift+P.